### ISSUE TITLE
Undefined Array Key

### ISSUE THREAD

**Issue by bhsmither** [cubecart/v6#3587]
In customers.index.inc.php, near line 332, the tested expression is to determine if **either** `$_GET['action']` **or** `$_POST['multi-action']` is set.

If the page was submitted via the drop-down selector (POST), as opposed to clicking on any of the action icons (GET), `$_GET` will not be set.

Thus, near lines 333, 340, 387, and possibly any hook included near line 499 will have PHP complain that `$_GET['action']` is not defined.

**Comment by bhsmither** [comment]
This solution seems to work:
```
Line 332, from:
if ( (isset($_GET['action']) || isset($_POST['multi-action'])) && Admin::getInstance()->permissions('customers', CC_PERM_EDIT)) {

To:
if ( (!empty($_GET['action'] ??= "") || isset($_POST['multi-action'])) && Admin::getInstance()->permissions('customers', CC_PERM_EDIT)) {
```

**Comment by TallFlash** [comment]
You're absolutely right! When the page is submitted via the drop-down selector (POST), $_GET['action'] won't be set, which is why PHP throws those undefined index notices. A simple fix could be to wrap the $_GET['action'] references with isset() checks to ensure it's only accessed when it's actually set. Something like:

```
php
if (isset($_GET['action'])) {
    // proceed with the logic
}
```
This should prevent PHP from complaining and make the code more robust when handling both GET and POST requests. Also, don't forget to apply the same logic for any other potential undefined variables. [CredibleBH](https://www.crediblebh.ltd/)
